src/ansys/aedt/core/generic/general_methods.py

```diff
@@ -905,10 +905,15 @@ def available_license_feature(
     if not input_dir:
         input_dir = list(aedt_versions.installed_versions.values())[0]
 
+    # The licensing client location has changed in the Windows 25R2 AEDT installation.
+    # Verification is required for Linux.
```

Comment:
I reviewed the code change and informational e-mail to confirm it applies to both OSs.
```suggestion
    # The licensing client location has changed in the 25R2 AEDT installation.
```

Reply:
Removed.